contracts/MemeFactory.sol

```diff
@@ -306,6 +378,9 @@ abstract contract MemeFactory {
         require(_locked == 1, "Reentrancy guard");
         _locked = 2;
 
+        // Check for name and symbol lengths
```

Comment:
This was already checked when shuffling functions, somehow the reference PR does not point to the correct one.